spec/factories.rb

```diff
@@ -1,2 +1,14 @@
 FactoryGirl.define do
+  factory :job do
+    position "World Class PHP Developer"
+    posting_date  3.days.ago
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.